nodejs-ruby-dev/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM        apiaryio/base-dev-nodejs
+MAINTAINER  Apiary <sre@apiary.io>
+
+ENV REFRESHED_AT 2015-10-07
+ENV RUBY_VERSION 2.2.3
+ENV BUNDLER_VERSION 1.10.6
+
+USER root
+
+# Install dependencies
+RUN apt-get update
+RUN apt-get install -y libreadline-dev libyaml-dev libxml2-dev libxslt-dev
+
+# Run install rbenv and ruby
+ADD script.sh /home/ubuntu/script.sh
```

Comment:
@freaz could you use COPY instead? (https://labs.ctl.io/dockerfile-add-vs-copy/)